docs/advanced.rst

```diff
@@ -518,4 +465,4 @@ Each log line contains this information:
 Hard drive provisioning for multiple Kolibri servers
 ----------------------------------------------------
 
-For the recommended procedure to install and provision several Kolibri server devices, refer to our :ref:`hard drive provisioning tutorial <provision>`.
```

Comment:
I'm confused: why this line appears as re-introduced?

Reply:
I honestly have no idea - I edited this directly in the github UI.